plugins/out_stackdriver/stackdriver.c

```diff
@@ -842,16 +875,7 @@ static int pack_json_payload(int operation_extracted, int operation_extra_size,
             continue;
         }
 
-        len = kv->key.via.str.size;
-        for (j = 0; j < len_to_be_removed; j++) {
-            removed = to_be_removed[j];
-            if (flb_sds_cmp(removed, kv->key.via.str.ptr, len) == 0) {
-                key_not_found = 0;
-                break;
-            }
-        }
-
-        if (key_not_found) {
+        if (bool_map[i] == TO_BE_REMAINED) {
```

Comment:
if ```to_remove``` equals zero, we might skip packing every object

Reply:
I think the logic to skip pack every object when `to_remove` equals zero was here https://github.com/fluent/fluent-bit/pull/2385/files#diff-0780c5bd4aa79e6c53b446ed0f6a6914R851. Am I understanding what you mean right?